cmd/onos-config/onos-config.go

```diff
@@ -70,7 +71,7 @@ func (i *arrayFlags) Set(value string) error {
 // The main entry point
 func main() {
 	var modelPlugins arrayFlags
-
+	allowUnvalidatedConfig := flag.Bool("-allowUnvalidatedConfig", false, "allow configuration for devices without a corresponding model plugin")
```

Comment:
Does making the flag start with "-" make it so that usage has to be "--allowUnvalidatedConfig" rather than "-allowUnvalidateConfig"? Just curious.

Reply:
Oops. Copy and paste error. Good catch. The idea is that if nothing is specified, then it will enforce validation against plugins. I hope the choice of name makes this clear

Reply:
Yes, the name is perfectly clear; I just wasn't entirely sure if the extra "-" had some other meaning to the flags package. :)